main/mail/stubs/dovecot.conf.mas

```diff
@@ -243,7 +243,7 @@ dict {
 ## Plugin settings
 ##
 plugin {
-  quota = maildir:User quota
+  quota = maildir
```

Comment:
"User quota" is the message returned to the IMAP client. Apple Mail crashes if empty (according to Dovecot wiki)